codeflash/optimization/optimizer.py

```diff
@@ -342,6 +343,11 @@ def run(self) -> None:
                 logger.info("❌ No optimizations found.")
             elif self.args.all:
                 logger.info("✨ All functions have been optimized! ✨")
+                response = send_completion_email()
```

Comment:
Q: Aren't we sending the mail with some details on optimization?
Or is it planned in TODO?

Reply:
So Can you please add it as a comment for TODO here.